src/products/mozilla_org/testing.md

```diff
@@ -52,28 +52,62 @@ You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.or
 * Test the page layout on the leading locally developed browser if available.
 * Test the page layout on mobile devices of major platforms.
 
-## When can I see the localized page on the production server?
+## Page activation
```

Comment:
The target here is localizers, I feel like "page activation" is internal jargon. Is there a reason not to keep the old title?

Reply:
I changed it back. This is the only title with a question mark. I couldn't find a better way to say it.

Reply:
How to test localized pages on the production server